### PR TITLE
Support dygraph quant model

### PR DESCRIPTION
**Pull request by juncaipeng**
### PR types

Others
### PR changes

Others
### Describe

Support dygraph quantized model.

Paddle 2.0 uses the dygraph model as default,  and the model quantized by Paddl2.0 is different from Paddle1.8.

In the dygraph quantized model, the inputs -> fake_quant_dequant_ops ->quantized_op. 

Besides, the weights is real float32. 

image

### CODE REVIEW

**Review by wzzju — DISMISSED** [review]
LGTM.

**Comment by juncaipeng** [comment]
@wojtuss  Please review this PR. Thanks.

File: python/paddle/fluid/contrib/slim/quantization/quant2_int8_mkldnn_pass.py [hunk 1]
```diff
@@ -290,10 +297,15 @@ def _swap_inputs(self, op, old_input, new_input):
                 ])
 
     def _dequantize_weights(self, graph):
+        def _is_int8_weights(op_node, weight_name):
+            weight_var_name = op_node.input(weight_name)[0]
+            weight = self._load_param(self._scope, weight_var_name)
+            return np.all(np.abs(weight % 1) < 1e-6)
```
Comment (wojtuss) [inline python/paddle/fluid/contrib/slim/quantization/quant2_int8_mkldnn_pass.py]: I guess the `1e-6` constant comes from the precision of fake-quantization. Can it be named and connected to the precision? I do not know how likely it is for the precision to change in the future.

Comment (juncaipeng) [reply python/paddle/fluid/contrib/slim/quantization/quant2_int8_mkldnn_pass.py]: The function analyzes the weights is real float32 or fake float32 (int8 range).
I have updated `np.all(np.abs(weight % 1) < 1e-6)`  to `np.all(np.mod(weight, 1) == 0)`.

**Review by luotao1 — APPROVED** [review]

File: python/paddle/fluid/contrib/slim/quantization/quant2_int8_mkldnn_pass.py (hunk 1 above)
Comment (juncaipeng) [reply python/paddle/fluid/contrib/slim/quantization/quant2_int8_mkldnn_pass.py]: @wojtuss   As the tag day is January 12 and the QA have to test the module, we merge the PR right now. If you have other comments, please let me know and I will fix in the next PR.

Comment (wojtuss) [reply python/paddle/fluid/contrib/slim/quantization/quant2_int8_mkldnn_pass.py]: @juncaipeng , Thanks. LGTM

**Pull request merged**